lib/modules/versioning/rpm/index.spec.ts

```diff
@@ -0,0 +1,174 @@
+import rpm from '.';
```

Comment:
I think you should create a `lib/modules/versioning/rpm/readme.md` file that explains what your `rpm` versioning does? For example, here's the Debian versioning readme:

https://github.com/renovatebot/renovate/blob/main/lib/modules/versioning/debian/readme.md